src/style.css

```
@@ -0,0 +1,37 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+ 
+.stars {
+  display: flex;
+}
+ 
+.stars__star {
+  background-image: url("../src/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+ 
+.stars--1 div:first-child {
```

```suggestion
.stars--1 .stars__star:first-child {
```
У этого элемента есть класс